lib/services/execution_iframe.dart

```diff
@@ -142,32 +167,35 @@ window.onerror = function(message, url, lineNumber, colno, error) {
       }
     });
 
-//    window.onMessage.listen((MessageEvent event) {
-//      Map data;
-//
-//      try {
-//        // TODO: This throws in Safari and FireFox with the polymer polyfills active.
-//        if (event.data is! Map) return;
-//        data = event.data;
-//        if (data['sender'] != 'frame') return;
-//      } catch (e) {
-//        print('${e}');
-//
-//        return;
-//      }
-//
-//      String type = data['type'];
-//
-//      if (type == 'stderr') {
-//        // Ignore any exceptions before the iframe has completed initialization.
-//        if (_readyCompleter.isCompleted) {
-//          _stderrController.add(data['message']);
-//        }
-//      } else if (type == 'ready' && !_readyCompleter.isCompleted) {
-//        _readyCompleter.complete();
-//      } else {
-//        _stdoutController.add(data['message']);
-//      }
-//    });
+    window.onMessage.listen((MessageEvent event) {
+      Map data;
+
+      try {
+        // TODO: This throws in Safari and FireFox with the polymer polyfills active.
```

Comment:
I suspect this code can go away, as we aren't using polymer polyfills

Reply:
Looks like it was added by @devoncarew, here:  https://github.com/dart-lang/dart-pad/commit/80a9e6aeef1123ed1bcedd5d6a9260069a3e30f2

Hopefully he can shed some further light on this.

Reply:
Unfortunately, I no longer have context on this, but I suspect that the comment is completely out of date, given how how much browsers / shadowdom / polymer support have changed since then.